scripts/setup.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
+echo "Allow Apache"
```

Comment:
para alinearlo con el resto de los mensajes, lo arrancaría todo con minúscula y lo expresaria como "allowing apache"

un nice to have: para diferenciar esto más facilmente del log de los comandos en sí, se podría agregar algo como

`----- allowing apache -----`